idx/memory/memory.go

```diff
@@ -242,17 +237,17 @@ func (m *MemoryIdx) add(def *schema.MetricDefinition) error {
 		log.Debug("memory-idx: creating branch %s with child %s", branch, nodes[i])
```

Comment:
are we talking about the same loop? actually that's really not important enough to spend time arguing, but i think according to your description you're referring to the previous loop.